acceptance/features/edit_branch_page_spec.rb

```diff
@@ -44,7 +44,16 @@
     when_I_save_my_changes
     then_I_should_be_on_the_correct_branch_page('edit')
 
-    then_I_can_add_and_delete_conditionals_and_expressions
+    then_I_can_add_conditionals_and_expressions
+
+    and_I_choose_an_option(
+      'branch[conditionals_attributes][0][expressions_attributes][0][component]',
+      'What is your favourite hobby?'
+    )
+    when_I_save_my_changes
+    then_I_should_not_see_an_error_summary
+
+    then_I_can_delete_conditionals_and_expressions
```

Comment:
just before we delete them please can we check that 'Which flavours of ice cream have you eaten?' and 'What is your favourite hobby?' are correctly populated on the first and second conditionals respectively